components/card.html

```diff
@@ -45,23 +51,35 @@ <h2>Card with Badge</h2>
         <p>
           A default image is displayed for avatars when a chosen source is unavailable (for example, when there is a problem displaying the image due to an error), or simply unspecified.
         </p>
-        
-        <div class="card">
-          <div class="child-card">
-            <img src="../components/images/chair.png" class="" />
-            <div class="badge-on-card">
-              #1 Bestseller
+
+        <div class="code-border">
+          <div class="code-bg">
+            <div class="card">
+              <div class="child-card">
+                <img src="../components/images/chair.png" class="" />
```

Comment:
Suggestion:- This is no alt attribute please add alt attribute. Class attribute is also empty.